openshift-serverless-clients.spec

```
@@ -28,7 +29,7 @@ Client kn provides developer experience to work with Knative Serving APIs.
 %setup -q -n %{source_dir}
 
 %build
-TAG=%{kn_cli_version} make build-cross
+TAG=%{kn_cli_version} KN_PLUGIN_EVENT_SENDER_IMAGE=%{kn_event_image} make build-cross
```

```suggestion
TAG=%{kn_cli_version} \
KN_PLUGIN_EVENT_SENDER_IMAGE=%{kn_event_image} \
make build-cross
```

For better readability